README.md

```diff
@@ -118,6 +118,28 @@ In this example, the Action will load the OIDC token from the GitHub-provided en
 ```
 In this example, the secret `AWS_ROLE_TO_ASSUME` contains a string like `arn:aws:iam::123456789100:role/my-github-actions-role`.  To assume a role in the same account as the static credentials, you can simply specify the role name, like `role-to-assume: my-github-actions-role`.
 
+```yaml
```

Comment:
We don't need a full example, but I'd like for there to be at least a description of `drop-current-credentials` in the README with a short explanation on when it would be useful to use